### ISSUE TITLE
NTR: Abnormal urine 3,5,6-trichloro-2-pyridinol level

### ISSUE THREAD

**Issue by laurenechan** [obophenotype/human-phenotype-ontology#7250]
**Preferred term label:**
Abnormal urine 3,5,6-trichloro-2-pyridinol level

**Synonyms**
abnormal urine TCPy level
abnormal urine TCP level

**Definition (free text, please give PubMed ID)**
Any deviation from the normal concentration of 3,5,6-trichloro-2-pyridinol in the urine.
PMID: 15367928 (investigation that measures this metabolite in urine)

**Parent term (use hpo.jax.org/app)**
Abnormal urine metabolite level (HP:0033354)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Associated with exposure to chlorpyrifos and subsequent sequelae of outcomes like cognitive disorders

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-7463-6306

**Comment by pnrobinson** [comment]
@laurenechan is this an endogenous metabolite or an exposure to a toxin?
If so should it go under here?
https://hpo.jax.org/app/browse/term/HP:0031840

**Comment by laurenechan** [comment]
@pnrobinson My understanding is that this is a metabolite produced following exposure to a toxin (chlorpyrifos), but it is not a direct exposure to this metabolite. It does not appear to be endogenously produced without the exposure to chlorpyrifos. With that in mind, do you think that the urine xenobiotic is the more appropriate parent class?

**Comment by pnrobinson** [comment]
@laurenechan Yes, this is a xenobiotic then. There is no use in having an Abnormal ..level class if the only relevant abnormality is "increased", so I would suggest
Label: Increased urine 3,5,6-trichloro-2-pyridinol level
Elevated amount of 3,5,6-trichloro-2-pyridinol in the urine.

In general, we do not use "concentration" for the urine terms, since the actual concentration is dependant onthe amount of urine, instead, we say level.

Could you also provide a comment with some explanation of why this is important?

**Comment by laurenechan** [comment]
The recommended label and definition make sense, thank you for the insight!

As for context of why this is an important metric, oral exposure to chlorpyrifos is often assessed using urine measurements of 3,5,6-trichloro-2-pyridinol. Estimating exposures of chlorpyrifos as well as evaluating any reported outcomes seen in exposed populations was what helped fuel the eventual banning of chlorpyrifos for both household and commercial use in the US. Here's an example of a study which leveraged this urine biomarker for estimating chlorpyrifos exposure in children https://www.nature.com/articles/7500406

**Comment by pnrobinson** [comment]
Adding as Elevated urine 3,5,6-trichloro-2-pyridinol level (there is no too low term!)
HP:0034296
